### PR TITLE
WIP: Initialization of Russian translation. Issue #87.

### PR DESCRIPTION
**Pull request by Ilyushin**

### CODE REVIEW

File: README-ru.md [hunk 1]
```diff
@@ -0,0 +1,2018 @@
+# Google Interview University
+
+Оригинальная  версия: [Английский](README.md) 
+
+## Что это?
+
+Это мой учебный план рассчитанный на несколько месяцев для веб-разарботчиков, не имеющих образования в CS и планирующих 
+работать разработчиками программного обеспечения в компании Google.
```
Comment (aminought) [inline README-ru.md]: Перевёл две строчки и уже pull request?

Comment (Ilyushin) [inline README-ru.md]: @aminought Ну так что бы люди двойную работу не делали, видели что переводят.

**Comment by niceyass7** [comment]
nerci

**Comment by jwasham** [comment]
When you're ready for me to pull, let me know.

**Comment by jwasham** [comment]
@Ilyushin and all, let me know when you're ready for me to merge. Thanks for all your contributions!

**Comment by contributorpw** [comment]
@jwasham I think this should be merge, as the fork already has a lot of changes.

**Comment by Ilyushin** [comment]
> @Ilyushin and all, let me know when you're ready for me to merge. Thanks for all your contributions!

@jwasham yes, sure, we have to finish a little bit. Thank you for your titanic labor!

**Comment by jwasham** [comment]
So merge now, or wait a bit?

**Comment by Ilyushin** [comment]
> So merge now, or wait a bit?

@jwasham Wait please, I'll let you know when it's ready.

**Comment by jwasham** [comment]
I'll pull this.

**Pull request merged**